src/network-config.js

```diff
@@ -0,0 +1,42 @@
+const getNetworkConfig = function getNetworkConfig(networkId) {
+    switch (networkId) {
+        case 'mainnet':
+            return {
+                nodeUrl: 'https://rpc.mainnet.near.org',
+                networkId: 'mainnet',
+                evmAccountId: 'evm',
+                walletUrl: 'https://wallet.near.org',
+                explorerUrl: 'https://explorer.near.org',
+            };
+        case 'testnet':
+            return {
+                nodeUrl: 'https://rpc.testnet.near.org',
+                networkId: 'default',
+                evmAccountId: 'evm',
+                walletUrl: 'https://wallet.testnet.near.org',
+                explorerUrl: 'https://explorer.testnet.near.org',
+            };
+        case 'betanet':
+            return {
+                nodeUrl: 'https://rpc.betanet.near.org',
+                networkId: 'betanet',
+                evmAccountId: 'evm',
+                walletUrl: 'https://wallet.testnet.near.org',
+                explorerUrl: 'https://explorer.testnet.near.org',
+            };
+        case 'local':
```

Comment:
Is it always specified everywhere as "local", or can it be sometimes "localnet"? I think I've seen the latter in some docs, but not sure.

Reply:
Good question. Indeed `nearup` creates files in `~/.near/locanet` which might be where you recognize that.
In terms of configuration, we use 'local' for this.
https://github.com/near/near-cli/blob/91a0cba74eb51709fa18568802ed36bccfaa2dcc/config.js#L38
and then all our examples with config, like for example:
https://github.com/near-examples/guest-book/blob/b4557f0b5eecffe522c0acc3ce75ac325a3ee2bb/src/config.js#L33